docs/mobile/game-session/game_session_db_design.md

```diff
@@ -0,0 +1,53 @@
+# DynamoDB Design
+
+## Entities
+
+### Config
+
+This entity contains the global settings and configurable parts of the application. The list of items included here is as follow
+
+- Points granted to each team estimated the right popular trick answer
+- Points granted per to teams guessed the right answer
+
+### GameSession
+
+The higher level entity contains all information related to a game from start to finish.
+
+- Game: A game object that comes from Aurora. It's a snapshot of the data when the game starts
+- Teams: An array of team entities that involved in the game
+- StartedDateTimeUTC: Date time of when the game started in UTC
+- MultiDeviceMode: Boolean indicating if it's a multi-device with facilitator or not
+
+### Team
+
+- TeamNo / TeamId: Numeric unique number assigned to the team
+- Team name: Name chosen to show fo the team. This helps for future substituting `team 1` string with chosen team names
+- Question: The chosen question for the team
+- GameCode: A 4-digit code to enter to join the team
```

Comment:
We use codes to join teams? I thought there was one code for the whole game.

Reply:
Though "No" is indeed a common abbreviation for Number, might TeamNum or something along those lines be worth considering?

Reply:
We talked about it, the idea here was assigning codes for each team so audience joins with a code that assigned them to a specific team. For now, that's not needed so moving this to `GameSession`